project/ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,109 @@
+* host distro ( ubuntu, RHEL, fedora )
+
+A bonus is to ask for a way to reproduce the issue.
+
+Additional information can be requested if needed.  If an issue is totally unable
+to be triaged based on the author's provided information, kindly state that
+they will have to provide the above information and clarification on the problem.
+Close the issue letting them know they can re-open once the information is 
+available so that it is able to be properly triaged.  Closing the issue is ok
+because some people do not have time to respond quickly and there is no use
+leaving an issue open that cannot be acted upon.  Feel free to point them to 
+this document so that they understand the issue is not being closed due to
+maintainers and contributors not wanting to help but because they don't have
+the information to provide help in a meaningful way.
+
+
+### Classification
+
+Classifications can come in a few different flavors.  These are used to help
+prioritize and understand what type of issue this is and how it should be 
+resolved. 
+
+Types of classification:
+* improvement - improvements are not bugs or new features but can drastically 
+improve usability.
+* bug - bugs are bugs.  The cause may or may not be known at triage time so
+debugging should be taken account into the time estimate. 
+* regression - regressions are usually easy fixes as hopefully the action
+worked previously and git history can be used to propose a solution.
+* feature - features are new and shinny.  They are things that the project does
+not currently support.
+
+
+### Time estimate
+
+Everyone who has ever written software knows how time estimates work.  Every PM
+has their own special formula for when a developer gives them an estimate the
+multiply it by a factor of 10 or something like that.  Every developer thinks 
+that you cannot rush greatness.  So why should an open source project place
+time estimates on issues?  Because this is not requiring that an issue should
+be completed in this amount of time by a contributor, it is being used as a
+way for the contributor to know how long thing may take if they wanted to work
+on this task.  Maybe someone feels like hacking on docker over the weekend and
+are looking for a weekend task.  Maybe someone has a couple hours at night as 
+they wind down and wants to help improve documentation or fix a few bugs.
+
+Estimates are a way for a contributor to know what they are getting into to
+make sure that contributing to docker is a positive experience.
+
+Types of estimates:
+* quick-fix - 30 minutes to a couple hours 
+* weekend-hack - depending on your knowledge of the code base this could take
+a few hours to create a successful pull request with tests included
+* sprint - you have some time to hack on a feature a few hours each day
+* marathon - this may take awhile as it could touch many aspects of the code and
+it maybe an issue where you need to discuss and plan on how it will be developed
+with others.
```

Comment:
Just a suggestion, but someone (maybe Jerome?) recently told me about a "belt" classification system (such as: "white-belt", "yellow-belt", "blue-belt", "black-belt"), which I think is a fun way to do things.

Plus, maybe I'm wrong, but I'd rather rate difficulty than time: a one-line change in `daemon/` takes as much time as one-line change in `cli/commands.go` but might require a better understanding of Docker as a whole to get it right (obviously that's a totally random example).